pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -134,6 +137,12 @@ type MonSpec struct {
 	AllowMultiplePerNode bool `json:"allowMultiplePerNode"`
 }
 
+type MgrSpec struct {
+	Count                int  `json:"count"`
```

Comment:
@liewegas or @leseb, are there situations where (1) more than 2 mgrs makes sense or (2) a single mgr is preferable to having two mgrs?

If more than two mgrs doesn't make sense and if one mgr doesn't make sense except for the case of single-node clusters with host networking enabled, I think it could be more user friendly to just always create two and not make them configure this in the crd.